nuxeo-features/nuxeo-thumbnail/src/test/java/org/nuxeo/ecm/platform/thumbnail/test/TestRecomputeThumbnails.java

```diff
@@ -123,4 +123,20 @@ public void testRecomputeThumbnails() throws Exception {
         assertNotNull(thumbnail);
     }
 
+    @Test
+    public void testThumbnailsForTIFF() throws Exception {
+
+        DocumentModel doc = session.createDocumentModel("/", "testDoc", "File");
+        Blob blob = Blobs.createBlob(FileUtils.getResourceFileFromContext("test-data/at3_1m4_01.tif"), "image/tiff",
+                StandardCharsets.UTF_8.name(), "at3_1m4_01.tif");
+        doc.setPropertyValue("file:content", (Serializable) blob);
+        doc = session.createDocument(doc);
+
+        // wait for thumbnail generation
+        txFeature.nextTransaction();
+        doc = session.getDocument(doc.getRef());
+        Blob thumbnail = thumbnailService.getThumbnail(doc, session);
```

Comment:
you can also add a second check to ensure that the expected type for this case is the one you configured in the contribution ` assertEquals("image/png", thumbnail.getMimeType());`